src/OrchardCore.Themes/TheAdmin/wwwroot/Scripts/TheAdmin.js

```diff
@@ -7090,7 +7106,11 @@ $(function () {
         // use a custom message if its set in data-message
         var title = $(this).data('title');
         var message = $(this).data('message');
```

Comment:
Why are we doing $(this) constantly instead of adding a var $this = $(this); ?

Reply:
Not really a big perf hit i guess: https://jsperf.com/caching-jquery-selectors

Reply:
We could review all the scripts in general before 1.0.0 release